examples/project-service-list-simple/.expected/diff.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/service_proj1-service-compute.yaml b/service_proj1-service-compute.yaml
+new file mode 100644
+index 0000000..b02d5d8
+--- /dev/null
++++ b/service_proj1-service-compute.yaml
+@@ -0,0 +1,10 @@
++apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
++kind: Service
++metadata:
++  name: proj1-service-compute
```

Comment:
#554 allows you to configure the namespace of the service. What is the best practice for using namespace in `Services`?

Reply:
Any annotations/namespace set for ProjectServiceList CR is added to all generated service resources. I have a section [here](https://github.com/bharathkkb/kpt-functions-catalog/tree/project-services/functions/go/project-service-list#:~:text=Sets%20namespace%20if%20any%20defined%20for%20ProjectServiceList%20custom%20resource%20to%20each%20generated%20resource.) on that. Users can choose to set a NS (recommended) or use default.